k2/csrc/rnnt_decode.cu

```diff
@@ -31,17 +31,10 @@ namespace k2 {
 namespace rnnt_decoding {
 
 std::shared_ptr<RnntDecodingStream> CreateStream(
-    const std::shared_ptr<Fsa> &graph) {
-  K2_CHECK_EQ(graph->shape.NumAxes(), 2);
-  ContextPtr &c = graph->shape.Context();
-  RnntDecodingStream stream;
-  stream.graph = graph;
-  stream.num_graph_states = graph->shape.Dim0();
-  // initialize to start state
-  stream.states = Ragged<int64_t>(RegularRaggedShape(c, 1, 1),
-                                  Array1<int64_t>(c, std::vector<int64_t>{0}));
-  stream.scores = Ragged<double>(stream.states.shape,
-                                 Array1<double>(c, std::vector<double>{0.0}));
+    const Fsa &graph) {
+  K2_CHECK_EQ(graph.shape.NumAxes(), 2);
+  ContextPtr &c = graph.shape.Context();
+  RnntDecodingStream stream(graph);
```

Comment:
This function is equivalent to 
```cpp
std::make_shared<RnntDecodingStream>(graph);
```
Is there a reason to create a new function instead of using
`std::make_shared<RnntDecodingStream>(graph);` directly?

Reply:
I think we can remove it.